cmd/minikube/cmd/version.go

```diff
@@ -29,8 +31,10 @@ var versionCmd = &cobra.Command{
 	Short: "Print the version of minikube.",
 	Long:  `Print the version of minikube.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		// Empty func to override parent pre-run - don't want to perform
-		// check for new version.
+		// Explicitly disable update checking for the version command
+		viper.Set(config.WantUpdateNotification, "false")
```

Comment:
We just discussed moving this to a package variable instead of going back through Viper.